Our /usr/bin/vncviewer is symlink to alternatives. This desktop file is named
specifically "TigerVNC Viewer", so lets start /usr/bin/vncviewer-tigervnc, no
matter what the currently selected alternative is.

Index: tigervnc-1.16.0/vncviewer/vncviewer.desktop.in.in
===================================================================
--- tigervnc-1.16.0.orig/vncviewer/vncviewer.desktop.in.in
+++ tigervnc-1.16.0/vncviewer/vncviewer.desktop.in.in
@@ -2,7 +2,7 @@
 Name=TigerVNC
 GenericName=Remote desktop viewer
 Comment=Connect to VNC server and display remote desktop
-Exec=@CMAKE_INSTALL_FULL_BINDIR@/vncviewer
+Exec=@CMAKE_INSTALL_FULL_BINDIR@/vncviewer-tigervnc
 Icon=tigervnc
 Terminal=false
 Type=Application
